src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -13633,6 +13620,7 @@ void Player_Action_8084FBF4(Player* this, PlayState* play) {
     func_8002F8F0(&this->actor, NA_SE_VO_LI_TAKEN_AWAY - SFX_FLAG + this->ageProperties->unk_92);
 }
 
+#if OOT_DEBUG
 s32 func_8084FCAC(Player* this, PlayState* play) {
```

Comment:
Was the call to this function removed as well? (How does this build?)

We should probably `#if OOT_DEBUG` the declaration too

Reply:
i did not handle the call to it yet, its being finicky to match. will post a scratch soon.
I can remove the ifdef for now until thats handled i guess